bot/mc.go

```diff
@@ -0,0 +1,43 @@
+package bot
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"venova/sshd"
+
+	"github.com/gorcon/rcon"
+)
+
+func restartMinecraft() {
+	client, err := sshd.ConnectToDev()
+	if err != nil {
+		log.Printf("SSH error: %v", err)
+	}
```

Comment:
we're leaking ssh connections right now

```suggestion
	}
	defer client.Close()
```